### DISCUSSION TITLE
resource安全性的具体例子

### DISCUSSION THREAD

**Discussion by livelybug** [NonceGeek/Move-Camp#160] [Q&A] [answered]
Move的一个特点是更高的资产的安全性，比如resource不能copy。

* 假如APTOS是POW，能防止51%双花攻击吗？
* 假如MOVE合约漏洞，能防止黑客mint token吗？

能不能举几个具体例子说明安全性。

**Comment by mason0510** [comment]
双花啥啊 这边不可能余额双花

**Comment by mason0510** [comment]
你都有漏洞了 赶紧把钱提出来跑路吧

**Comment by leeduckgo** [comment, marked as answer]
一个很好的例子是跨链时候 Resource 模型更方便映射。用户只要把其 Resource 映射到另一条链上就行了，而以太坊的合约内账本映射更加复杂。
